internal/proxy/usecases/proxy_usecase.go

```diff
@@ -70,6 +73,22 @@ func (p *ProxyUsecase) findInDatabaseIfNotFoundInCache(UUID string) error {
 	return nil
 }
 
+// WriteCachedRequestsRoutine  write metric cache in database
+func (p *ProxyUsecase) WriteCachedRequestsRoutine() {
+	logrus.Info("func WriteCachedRequestsRoutine")
+	allRequests, err := p.lruMetrics.GetAll()
+	if err != nil {
+		logrus.Error("could not init the LRU cache")
```

Comment:
We need the error, I suggest : `logrus.Errorf("could not get cached metrics: %s", err)`